src/contact-summary/contact-summary-emitter.js

```diff
@@ -52,6 +63,12 @@ function emitter(contactSummary, contact, reports) {
   return result;
 }
 
+function convertToArray(appliesToType) {
+  return Array.isArray(appliesToType) ? 
+                        appliesToType : 
+                        (appliesToType ? appliesToType.split() : [undefined]);  
```

Comment:
consider just `Array.isArray(x) ? x : [x]` instead of doing this conditional evaluation with the split